codecov/gazebo
# Assortment of UI changes for coverage over time for components

# Description

fixes: https://github.com/codecov/engineering-team/issues/1639

- Add a link to redirect from components tab on PR page to components tab on Repo page
- Add latest commit under the branch selector
- Disable the branch dropdown box on the not_configured/enable/backfilling views
- When clicking "Enable component analytics" button it should reload the page (which would bring it to backfilling view)
- Fix padding on the dropdowns section. The padding between the vertical line and dropdown and padding between the dropdown label and dropbown
- Add table headers when showing the not_configured/enable/backfilling views
- Last Uploaded table header row values should align to the left
- The delete component modal message is not correct
- Add a aria-label = "delete-{component_id}" to the trash can icon to delete
- Update the message from 24 hour to 1 hour on the backfilling view

# Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. In 2022 this entity acquired Codecov and as result Sentry is going to need some rights from me in order to utilize my contributions in this PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.
